pyEpiabm/pyEpiabm/toy_population_config.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+from .population import Population
+from .household import Household
+
+
+class ToyPopulationFactory:
+    """ Class that creates a toy population for use and returns in the simple
+    python model.
+    """
+    def make_pop(self, population_size: int, cell_number: int,
+                 microcell_number: int, household_number: int,
```

Comment:
`Household_number` should probably have a default value so it is an optional variable, given that it is meaningless if they user selects `if_households = False` (as indeed is the default).

I'm not sure what this value should be though, maybe 1 - (i.e. everyone is in the same household?) I guess the natural choice would `population_size` but I am not sure you can set the default value equal to another variable?

Reply:
This has been documented in issue #26 and will not be resolved here